lib/cosmos/system/system.rb

```diff
@@ -996,7 +996,15 @@ def build_cmd_system_meta
         # in CommandSender since they don't have write conversions due to:
         # commands.build_cmd -> packet.restore_defaults -> packet.write_item
         next if Packet::RESERVED_ITEM_NAMES.include?(item.name)
-        cmd_meta.define(item.clone)
+        item = cmd_meta.define(item.clone)
```

Comment:
This does not handle array types.  Need to check for array and just put an empty array as well.